_sections/30-projects.md

```diff
@@ -18,4 +18,5 @@ style: center
 - [QIIME](http://qiime.org)
 - [gala](https://gala.readthedocs.io)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [xonsh](https://xon.sh)
```

Comment:
xonsh can go quite a bit higher, using the prominence-as-indicated-by-github-stars metric: it's between sympy and scikit-bio.